solution/ui/regulations/js/src/components/SupplementalContent.vue

```diff
@@ -115,15 +125,24 @@ export default {
     },
 
     methods: {
-        async fetch_content(title, part) {
+        async fetch_content() {
```

Comment:
if removing parameters for this method, then `this.title` and `this.part` arguments can be removed from method invocation on line 105